src/main/java/lesson5/selenium/GlobalSettings.java

```diff
@@ -0,0 +1,10 @@
+package lesson5.selenium;
+
+import static java.lang.System.getProperty;
+
+public class GlobalSettings {
+
+    public final static String START_URL = "https://pzz.by/";
+    final static String DRIVER_PATH = "target/drivers/chromedriver.exe";
```

Comment:
you have resources folder for all of your resources. No need to directly point to target directory. Moreover, prefer loading files from classpath instead of specific location